Build.mak

```diff
@@ -26,4 +26,10 @@ $B/overview: $O/libvibed.a $O/libdyaml.a
 $B/overview: $C/src/overview/main.d
 $B/overview: override LDFLAGS += -L$O -lvibed -ldyaml -levent -lssl -lcrypto
 
-all += $B/overview
+$B/neptune: override LDFLAGS += -L$O -lvibed -levent -lssl -lcrypto
+$B/neptune: $O/libvibed.a
+$B/neptune: $C/src/neptune/main.d
+
+neptune: $B/neptune
```

Comment:
Actually I think calling neptune the tool to manage the releases makes sense. The overview tool is more like a hack IMHO and is not something that regular users will use normally. I think we can call this tool just `neptune` and other tools `neptune-whatever`. Or maybe we can go the "subcommands" route and have `neptune overview` and `neptune release` for example.